portfolio/src/main/webapp/projects.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Projects</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+      <div class = "content">
+          <h1> Past Projects </h1>
+          <nav>
+          <ul id = "horizontal-list">
```

Comment:
To avoid duplicating this navigation section on every page, you might want to include it dynamically:

https://stackoverflow.com/questions/31954089/how-can-i-reuse-a-navigation-bar-on-multiple-pages

(this is getting a bit advanced though, duplicating it is fine for now)